scaladoc/test/dotty/tools/scaladoc/site/SidebarParserTest.scala

```diff
@@ -38,10 +38,9 @@ class SidebarParserTest:
       |          - page: my-page6/my-page6/my-page6.md
       """.stripMargin
 
-  private val sidebarNoTitle =
+  private val sidebarErrorNoTitle =
     """index: index.md
       |subsection:
-      |  - title: My title
       |    page: my-page1.md
```

Comment:
This change makes the YAML invalid, I think there should be a `-` before `page`.

Also, I am surprised that the regression occurred only on Windows. Do you know why?